include/arch/arc/v2/irq.h

```diff
@@ -92,9 +92,6 @@ extern void z_irq_spurious(void *unused);
 
 static inline void arch_isr_direct_header(void)
 {
-#ifdef CONFIG_TRACING
```

Comment:
shouldn't this be changed to `CONFIG_TRACING_ISR` instead of completely removed? "direct" interrupts skip the common interrupt handling code AFAIK

Reply:
yes